app/tests/archives_tests/test_views.py

```diff
@@ -1458,3 +1458,24 @@ def test_item_job_list_row_template_ajax_renders(client):
     assert response_content["recordsTotal"] == 1
     assert len(response_content["data"]) == 1
     assert job_details_url in response_content["data"][0][0]
+
+
+@pytest.mark.django_db
+def test_interfaces_list_link_in_new_interface_form(client):
+    ar = ArchiveFactory()
+    u = UserFactory()
+    ar.add_editor(u)
+
+    response = get_view_for_user(
+        viewname="archives:item-new-interface-create",
+        client=client,
+        method=client.get,
+        reverse_kwargs={
+            "slug": ar.slug,
+        },
+        user=u,
+    )
+    assert (
+        reverse("components:component-interface-list-archives")
+        in response.rendered_content
+    )
```

Comment:
Any specific reason why you didn't parametrize this test (one test for both reader studies and archives)?

Reply:
ah, good point. I was thinking of keeping each test in its own app. Should I then move this test to the components app and cover both cases in a parametrized test?

Reply:
Yes, that sounds like a good place. I think we have a bunch of parametrized tests for archive items and display sets there as well.